src/closedWindows/closedWindowList/closedWindowList-controller.js

```diff
@@ -15,8 +19,29 @@
             this.refreshClosedWindows();
             this.closedTabsShow = this.closedWindows.length > 0;
         }
+
+        _watch() {
+            var listener = () => this.$timeout(() => this.refreshClosedWindows());
+            var addListener = () => {
+                window.storeService.addClosedWindowListener(listener);
+                this.refreshClosedWindows();
+            };
```

Comment:
Because this point is the earliest point that you can be certain `window.storeService` exists, it makes sense to load in the correct data from the service ASAP